inaugurator/partitiontable.py

```diff
@@ -9,8 +9,9 @@ class PartitionTable:
     _DEFAULT_SIZES_GB = dict(
         smallSwap=1,
         bigSwap=8,
```

Comment:
what is the meaning of swap? How do we use it (It's not part of the CR).
@eliran-stratoscale? @shimshon-stratoscale ?